src/wasm-traversal.h

```diff
@@ -49,6 +49,7 @@ struct Visitor {
   ReturnType visitSetGlobal(SetGlobal* curr) {}
   ReturnType visitLoad(Load* curr) {}
   ReturnType visitStore(Store* curr) {}
+  ReturnType visitAtomicRMW(AtomicRMW* curr) {return ReturnType();} //Stub impl so not every pass has to implement this yet.
```

Comment:
The return type is necessary because without it, non-void-returning visitors won't compile unless they have their own override. Probably all of these templates should have the `return ReturnType()` instead of being empty, otherwise passes can't fall back on them, which I'm guessing was not the intention.